pkg/slices.go

```diff
@@ -0,0 +1,14 @@
+package pkg
+
+type SliceFlags []string
+
+// provide a String() method on the type so we can use it with flag.Var
+func (i *SliceFlags) String() string {
+	return ""
+}
+
+// provide a Set() method on the type so we can use it with flag.Var
+func (i *SliceFlags) Set(value string) error {
```

Comment:
This can't be append

Reason:

```
./main.go:201:11: cannot use &f.filePaths (value of type *"github.com/kevincobain2000/gol/pkg".SliceFlags) as flag.Value value in argument to flag.Var: *"github.com/kevincobain2000/gol/pkg".SliceFlags does not implement flag.Value (missing method Set)
```

Reply:
indeed, in such case, I always try to add a comment like:

// required to implement flag.Value interface